pyapid/api.py

```diff
@@ -7,8 +7,11 @@
 from fastapi import FastAPI, HTTPException
 from starlette.status import HTTP_201_CREATED, HTTP_404_NOT_FOUND, HTTP_400_BAD_REQUEST
 from pyapid.types import Data, PyapidParseError
-import pyapid.reports as reports
+from pyapid import reports
+from pyapid.config import config
 
+# Call config to make sure we are caching the values in the very begining
+config()
```

Comment:
This stuff should probably be the only stuff in this one, the rest should probably be in some other PR as I doubt it has anything to do with the config cache thing.